### ISSUE TITLE
2x2 recipe identified with redundant resources in grid

### ISSUE THREAD

**Issue by AKryukov92** [DreamBlocks/DreamBlocks#25]
On the picture you can see crafting grid with three planks.
![image](https://user-images.githubusercontent.com/24266616/27509260-807361d0-591a-11e7-9ba0-4d9835dce457.png)
Game identified this recipe as recipe of sticks.
I think we'll get all kinds of troubles when we add more 2x2 recipes.
Assuming that template comparison process is very complicated, at this moment I file a report for future fix.

**Comment by LordStephen77** [comment]
@AKryukov92 Yes you are right. Unfortunately inventory classes, hotbar, I did not write them. They are very complicated classes because they design the ui by code, they would be allegory by applying images to better manage inventory and hotbar.

Same thing for crafting! My idea is to imitate Minetest crafting schemes. Obviously, creating a new "com.lordstephen77.dreamblocks.craftingsystem" package, all within the crafting management classes and recipes divided by category. Everything so much nicer so! As soon as I have time I try to write something.

If there are problems let me know. I almost finished the website, we will soon use it to recruit new collaborators and simplify our development.

Thank you for your help.
